authentication-request-handler/pom.xml

```diff
@@ -0,0 +1,170 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The file is indented with 8 characters, where WSO2 standard is 4. Can please fix it?

Reply:
Fixed with ef2449582ec210c80807bb2f728fec586f3846f2